Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabSaveLoadHandler.cpp

```diff
@@ -129,13 +143,242 @@ namespace AzToolsFramework
             settings->m_saveLocation = path;
         }
 
-        void PrefabSaveHandler::HandleSourceFileType(AZStd::string_view sourceFilePath, AZ::EntityId parentId, AZ::Vector3 position) const
+        // Drag and drop overrides:
+        int PrefabSaveHandler::GetDragAndDropEventsPriority() const
+        {
+            return 10; // higher than 0 so this class gets ahead of other built in 
+        }
+
+        // viewport dragging:
+        void PrefabSaveHandler::DragEnter(QDragEnterEvent* event, [[maybe_unused]] AzQtComponents::DragAndDropContextBase& context)
+        {
+            if (event->isAccepted())
+            {
+                return;
+            }
+
+            if (CanDragAndDropData(event->mimeData()))
+            {
+                event->accept();
+                event->setDropAction(Qt::DropAction::CopyAction);
```

Comment:
it tells the sending application (if its listening) that it should not MOVE the data into the viewport, just "copy it"

An example.  You open windows explorer, and you drag your C:\Windows folder onto an application that expects files and folders to be dropped.  When windows explorer picks it up and hovers it over your app, it 'hints' that it should be a MoveAction instead of a copy Action because the user didn't hold down the modifier key to indicate its a copy.

If your application accepts the action and accepts that its a MOVE action instead of changing it to a copy action, windows explorer _deletes c:\windows_ because the folder was dropped onto an appliation that 'accepted' the move.

THis is why its very important for applications that accept drop actions to specify whether they are accepting them as a copy, or as a move.